django_mongoengine_filter/widgets.py

```diff
@@ -19,10 +19,17 @@
 try:
     from django.utils.encoding import force_text
 except:  # pragma: nocover
-    from django.utils.encoding import force_unicode as force_text  # noqa
+    try:
```

Comment:
I think we can simply replace this whole thing with a single `force_str` import, since it doesn't make sense to support Django versions prior 2.2 (which is already quite generous). So, let's just import `force_str` and replace its' usages everywhere. Same goes for `gettext` import. We don't have to support older versions. Just import `gettext` as `_` and that's it.

Reply:
Ok I will implement that.

I found another issue in BaseFilterSet.__init__() function. I will include the necessary changes in the next pull request!